netcore/excludes-System.Runtime.Extensions.Tests.rsp

```diff
@@ -0,0 +1,20 @@
+# Environment.FailFast is not implemented
+-nomethod System.Tests.EnvironmentTests.FailFast_ExceptionStackTrace_InnerException
+-nomethod System.Tests.EnvironmentTests.FailFast_ExceptionStackTrace_ArgumentException
+-nomethod System.Tests.EnvironmentTests.FailFast_ExceptionStackTrace_StackOverflowException
+
+# Mono expands generic Types in stacktraces unlike the .NET Core (sounds more like a feature)
+-method System.Tests.EnvironmentStackTrace.StackTraceTest
+
+# AppContext.FirstChanceException event is never fired
+-nomethod System.Tests.AppDomainTests.FirstChanceException_Called
+
+# AssemblyLoadContext.InternalLoad is not implemented
+-nomethod System.Tests.AppDomainTests.LoadBytes
+
+# TODO: investigate
+-nomethod System.Tests.AppDomainTests.TypeResolve
+-nomethod System.Tests.AppDomainTests.GetAssemblies
+-nomethod System.Tests.AppDomainTests.ResourceResolve
+-nomethod System.Tests.AppDomainTests.AssemblyLoad
```

Comment:
The events are not wired up from the native side. In .NET Core the `AppDomain` class forwards them to `AssemblyLoadContext`. Mono on the native side still expects `AppDomain` (renamed to `MonoDomain` for .NET Core) which implements none of the events: https://github.com/mono/mono/blob/7082845114f603b3e174c26d9e8fb7f667bc2a51/mcs/class/System.Private.CoreLib/Mono/MonoDomain.cs#L17-L37

Reply:
Ah ok, so it's basically the same issues as with FirstChanceException event

Reply:
Yep. I was thinking about fixing it, but I didn't want to step on anyone's toes. I believe @lambdageek was doing some work on `AssemblyLoadContext`, perhaps not directly related to this?

Reply:
Yea, it's slow going. I got bogged down in refactoring `MonoImage:references` (a list of mono assemblies) to work correctly with ALCs.  I'll try to handoff the work to someone else. but right now it's stuck in a refactoring swamp

Reply:
This part seems to be quite isolated from the rest of `AssemblyLoadContext` in terms of dependencies. Aside from placing it in the right place in the native code it should not clash much with the rest of the refactoring.